src/main.rs

```diff
@@ -528,6 +528,15 @@ mod backtrace {
     }
 }
 
+fn play_scream() -> Option<()> {
+    const SCREAM: &[u8] = include_bytes!("data/R2D2-Scream.ogg");
+    let device = rodio::default_output_device()?;
+    let source = rodio::Decoder::new(std::io::Cursor::new(SCREAM)).ok()?;
+    rodio::play_raw(&device, rodio::source::Source::convert_samples(source));
+    std::thread::sleep(std::time::Duration::from_millis(2835));
```

Comment:
Is there no way to tell the length of the audio from `rodio`?

Reply:
Also, I'm undecided whether we should actually wait for it to finish, I think it might be funny to let the audio play while you read the panic.

Reply:
There is but it doesn't work